index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>How to Upload Files with JavaScript</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" type="text/css" href="./src/styles.css">
+</head>
+
+<body>
+<div id="app">
+  <h1>Awesome File Uploader 🦄</h1>
+  <div id="dropArea">
+  <form
+          action="https://httpbin.org/post"
+          enctype="multipart/form-data"
+          method="post"
+  >
+    <p>
+      Drag and Drop Your Files Here<br /><br /><span class="bold">or</span>
+    </p>
+    <input name="file" id="uploaderInput" multiple type="file" accept=".jpg, .png" />
```

Comment:
Just a little note - `accept` isn't a strict validation, it only suggests preferable types to the browser and OS. User will be able to choose anything he wants.

![image](https://github.com/tenebricosa/uploading-file-with-js/assets/670959/ddf4ca63-6867-433a-9b1e-879a2043ae17)0fc60ba299cd)

Reply:
Added JS validation as well!